package uz.uzkassa.smartpos.feature.company.saving.data.exception

internal class CompanyAlreadyCreatedException : Exception()